docs/pages/jpscore/jpscore_inifile.md

```diff
@@ -0,0 +1,303 @@
+    See [event_file](jpscore_events.html).
+    
+-   `<schedule_file>schedule.xml</schedule_file>`
+    The name and location of the schedule file. Schedules can be used to group doors and open or close this 
+    groups of door at certain points of time. See [schedule_file](jpscore_schedule.html)
+
+-   `<show_statistics>true</show_statistics>` Show different aggregate statistics e.g. the usage of the doors. (default: false)
+
+-   `<logfile>log</logfile>` save relevant information about the simulation to a log file with txt format.
+  Useful to keep track of warnings or errors that may rise during a simulation.
+
+
+-  The trajectory file
+
+```bash
+ <trajectories format="xml-plain" fps="8" color_mode="velocity">
+    <file location="trajectories.xml" />   
+ </trajectories>
+```
+
+The options for the format are
+
+-   `xml-plain`: the default xml format. It can lead to large files. See section [xml-plain](jpscore_trajectory#xml).
+
+-   `plain`: simple text format. See section [plain](jpscore_trajectory#txt).
+    - `optional_output`: possibility to give additional output. Set any of these values to `TRUE` (not case sensitive) to get the according value. `FALSE`, any other value and ignoring the option will lead to no output.
+        - `speed`: speed of the pedestrian
+        - `velocity`: x,y components of the pedestrians's velocity
+        - `final_goal`: id of the final goal the pedestrian is heading to
+        - `intermediate_goal`: id of the current goal the pedestrian is heading to (usually a door)
+        - `desired_direction`: x,y components of the pedestrians's desired directions
+        - `group`: group of the pedestrian
+        - `router`: router used by the pedestrian (id accoriding to ini-file)
+        - `spotlight`: pedestrian is highlighted.
+        
+        ```xml
+            <trajectories format="plain"  fps="8">
+              <file location="bottleneck_traj.txt" />
+              <optional_output speed="FALSE" velocity="TRUE" final_goal="NoOutputWrongValue"  intermediate_goal="false" desired_direction="true" group="TRUE" router="TRUE" spotlight="TRUE"/>
+            </trajectories>
+        ```
+        
+{%include warning.html content="xml trajectories tend to become large and hence slow down parsing and visualisation." %}
+
+- The value `fps` defines the frame rate per second for the trajectories.
+
+    - `color_mode`: coloring agents in the trajectories. Options are:
+       - `velocity` (default): color is proportional to speed (slow --> red).
+       - `spotlight`
+       - `group`: color by group
+       - `knowledge`
+       - `router`
+       - `final_goal`
+       - `intermediate_goal`
+
+    - `file location` defines the location of the trajectories.
+       All paths are relative to the location of the project file.
+
+
+## Train constraints
+
+Interface to trains is documented [here](jpscore_trains.html).
```

Comment:
Why are some things explained in external files, e.g., trains, operational models, and routers?

Reply:
I think because these are "big" topics and were outsourced to not overly make inifile-doc packed with information. 
I suggest doing the same for other important topics like "waiting areas"  and in general to all the topics that have some "logic" that needs documentation.

Reply:
Okay